### DISCUSSION TITLE
Surviv's settings

### DISCUSSION THREAD

**Discussion by Axmaz0** [IceHacks/SurvivCheatInjector#835] [Ideas]
When i first launched the game (sci icehacks app) i noticed that my language is my pc's language (dutch), which is a problem because theres no settings for the surviv.io game, [language, appearance, etc...] i really need it,
could you put that in the app back, because the setting is really necessary.

**Comment by IceHacks** [comment]
I have seen screenshots of some people's games where it is in Russian/German. So there is a way. However, I don't know of it.

**Reply by minhtq3-a11y** [reply]
> I have seen screenshots of some people's games where it is in Russian/German. So there is a way. However, I don't know of it.

can you share it?

**Reply by Axmaz0** [reply]
I mostly focused on the performance language, because some words i dont know yet in dutch but more in english.
